src/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@
 
 include(${UMF_CMAKE_SOURCE_DIR}/cmake/helpers.cmake)
 
+# set UMF_PROXY_LIB_BASED_ON_POOL to one of:
+# - SCALABLE
+# - JEMALLOC
+set(UMF_PROXY_LIB_BASED_ON_POOL SCALABLE)
```

Comment:
Do you want to enable setting this from the CLI level? I think you'd have to add `CACHE` (ref. https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry)

Reply:
Done